src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -308,15 +308,13 @@ public int getBlockPower()
 	@Override
 	public boolean isEmpty()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.getType().equals(Material.AIR);
 	}
 
 	@Override
 	public boolean isLiquid()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.getType().equals(Material.LAVA) || this.getType().equals(Material.WATER);
```

Comment:
Enum comparisons should be done using == instead of .equals() as it is faster and safer since it can do premature exclusions at compile time instead of runtime. Plus you get the benefit of avoiding to run into NullPointerException.

Reply:
Ah, I looked over this. Thanks for noticing!